### PR TITLE
Docs 6650-generate-scaffold-for-update-api-spec-ie

### PR DESCRIPTION
**Pull request by IsaacEldridge**
@fermujica I created the new topic, but added info about manually generating a flow.

### CODE REVIEW

File: modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc [hunk 1]
```diff
@@ -7,14 +7,14 @@ You can edit an imported API specification in your project as an API specificati
 
 Before you begin, ensure that you have the necessary permissions to edit the API specification in Design Center, and that you don't already have an API specification project for that API specification in your Studio workspace.
 
-. In the Package Explorer view, right-click the API dependency of your API specification project and select the name of the API you want to edit >  *Import API Spec from Design Center*.
+. In the Package Explorer view, right-click the API dependency of your API specification project, select the name of the API you want to edit, and click  *Import API Spec from Design Center*.
```
Comment (fermujica) [inline modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc]: > , and click

should be 

> , and select

File: modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc [hunk 2]
```diff
@@ -24,8 +24,7 @@ image::update-asset-version.png[]
 +
 image::update-version-api-dc.png[]
 . Select the new version of your API specification and select *Apply and Close*.
-+
-Studio updates the API specification to the selected version and offers you to scaffold the flows again based on the new version.
-
 
+Studio updates the API specification to the selected version, and asks you if you want to regenerate the xref:4.x@apikit::apikit-4-raml-flow-concept.adoc[flow] based on the new version.
```
Comment (fermujica) [inline modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc]: We must avoid inline links as much as possible.
The concept of a flow is something that the Studio audience is already familiar with, so it's not necessary here. 

> and asks you if you want to regenerate the xref:4.x@apikit::apikit-4-raml-flow-concept.adoc[flow] based on the new version.

Should be:

> and asks you if you want to regenerate the flow based on the new version.

File: modules/ROOT/pages/sync-update-api-spec.adoc [hunk 3]
```diff
@@ -0,0 +1,22 @@
+= Update the API Specification in Your Project
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+When necessary you can update the API specification in your Studio project.
```
Comment (fermujica) [inline modules/ROOT/pages/sync-update-api-spec.adoc]: I would change this line as something like:

> When you import an API specification as a dependency, you can update the API specification version in your Studio project.

Only APIs imported as dependencies from a Maven repository, or Exhcange can be updated like this. 
API specs that are manually imported from the file system cannot be updated.

File: modules/ROOT/pages/sync-update-api-spec.adoc [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+. In the Package Explorer view, right-click the Mule project that contains the API specification you want to update.
+. Select *API Specs* > the name of your API > *Update Version*.
++
+After you update the specification, Studio asks you if you want to regenerate the flows based on the new version.
++
+To manually generate a flow for the updated API, right-click the Mule project where you updated the API specification, and select *API Specs* > the name of your API > *Generate Flows*.
+
+Additionally, you can manage the API specifications in your project that are linked to Exchange from the API specification project management view. To do so, click the *Manage Mule Project APIs* icon (image:manage-project-api-icon.png[2%,2%]) in the top task bar.
+
+image::update-api-version.png[]
```
Comment (fermujica) [inline modules/ROOT/pages/sync-update-api-spec.adoc:20]: There is a padding missing in this screenshot.

File: modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc (hunk 2 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc]: Got it.

File: modules/ROOT/pages/sync-update-api-spec.adoc (hunk 4 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/sync-update-api-spec.adoc:20]: uploaded new image

File: modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc (hunk 2 above)
Comment (IsaacEldridge) [reply modules/ROOT/pages/sync-imported-api-specifications-design-center.adoc]: Got it.

**Review by fermujica — APPROVED** [review]

**Pull request merged**